ipn/ipnstate/ipnstate.go

```diff
@@ -132,8 +132,9 @@ type PeerStatus struct {
 	LastHandshake  time.Time // with local wireguard
 	Online         bool      // whether node is connected to the control plane
 	KeepAlive      bool
-	ExitNode       bool // true if this is the currently selected exit node.
-	ExitNodeOption bool // true if this node can be an exit node (offered && approved)
+	ExitNode       bool       // true if this is the currently selected exit node.
+	ExitNodeOption bool       // true if this node can be an exit node (offered && approved)
+	ExitNodeIP     netaddr.IP // exit node IP; assume zero value means exit node is unset
```

Comment:
I just realized that this is a little weird that this is an IP address, because which? IPv4? IPv6? other ones in the future when we do "TailNICs" letting a node have multiple identities.

At least document this as being "one of the IPs of the exit node"?  Probably time for a non-same-line comment.

```
// ExitNodeIP is set to one of the IP addresses of the exit node when an exit
// node is in use.
//
// It's set to the zero value when either an exit node is not in use, or the selected exit node is no longer present in the netmap.
```